src/pages/Post/Post.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable prettier/prettier */
+import './Post.css'
+
+import axios from 'axios'
+import React, {useEffect, useState } from 'react'
+
+interface IPost {
+    id: number,
+    links: {id: number, caption: string, url: string}[],
+    caption: string, 
+    short_text: string, 
+    details: string,
+    added_at: string,
+    show_after: string,
+    disable_after: string
+}
+
+const defaultPosts: IPost[] = []
+
+export const Posts: React.FC = () => {

```

Comment:
asi zacneme destrukturovat aj `FC` uz rovno v importe reactu
```suggestion
export const Posts: FC = () => {
```